### PR TITLE
Patch for MDC2020ae. Can't use an empty string variable in POMS

### PR DESCRIPTION
**Pull request by oksuzian**

### CODE REVIEW

**Comment by kutschke** [comment]
@oksuzian Does this need to get merged and a new Musing made before you can start your next stage?

File: CampaignConfig/mdc2020_corsika.ini [hunk 1]
```diff
@@ -6,14 +6,14 @@ campaign_stage_list = generation_par, generation, resampler_par, resampler, spil
 
 [campaign_defaults]
 vo_role=Production
-campaign_keywords={"s1type": "All", "testid": ""}
+campaign_keywords={"s1type": "All", "testid": "prod", "desc": "MDC2020ae"}
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_corsika.ini:9]: I thought testid was supposed to be null for production.  Will this value 'prod' propagate into collection names?

Comment (oksuzian) [reply CampaignConfig/mdc2020_corsika.ini:9]: POMS does not like null. 'prod' will not propagate into collection names.

**Comment by oksuzian** [comment]
> @oksuzian Does this need to get merged and a new Musing made before you can start your next stage?

@kutschke, Ray has already patched the version on cvmfs, and I'm running the production now.

**Review by brownd1978 — APPROVED** [review]

**Pull request merged**